webviz_subsurface/plugins/_multiple_regression.py

```diff
@@ -0,0 +1,1129 @@
+            )
+            scores_with_candidates.append((r_2_adj, candidate))
+
+        # If the best parameter is interactive, add all base features
+        scores_with_candidates.sort(key=lambda x: x[0])
+        best_new_score, best_candidate = scores_with_candidates.pop()
+        if current_score < best_new_score:
+            if " × " in best_candidate:
+                for base_feature in best_candidate.split(" × "):
+                    if base_feature in remaining:
+                        remaining.remove(base_feature)
+                    if base_feature not in selected:
+                        selected.append(base_feature)
+
+            remaining.remove(best_candidate)
+            selected.append(best_candidate)
+            current_score = best_new_score
+
+    # Finally fit a statsmodel from the selected parameters
+    model_df = data.filter(items=selected)
+    model_df["Intercept"] = np.ones((len(response), 1))
+    model_df["response"] = response
+    return _model_warnings(model_df)
+
+
+def _model_warnings(design_matrix: pd.DataFrame):
+    with warnings.catch_warnings():
+        # Handle warnings so the graphics indicate that the model failed for the current input.
+        warnings.filterwarnings("error", category=RuntimeWarning)
+        warnings.filterwarnings("ignore", category=UserWarning)
+        try:
+            model = sm.OLS(design_matrix["response"], design_matrix.drop(columns="response")).fit()
+        except (RuntimeWarning) as error:
+            print("error: ", error)
+            return None
+    return model
+
+
+def make_p_values_plot(p_sorted, theme, colors):
+    """ Make p-values plot """
+    p_values = p_sorted.values
+    parameters = p_sorted.index
+    fig = go.Figure()
+    fig.add_trace(
+        {
+            "x": [param.replace(" × ", "<br>× ") for param in parameters],
+            "y": p_values,
+            "type": "bar",
+            "marker": {
+                "color": [
+                    colors["default color"] if val < 0.05 else colors["gray"] for val in p_values
+                ]
+            },
+        }
+    )
+    fig.update_traces(
+        hovertemplate=[
+            "<b>Parameter:</b> "
+            + str(param)
+            + "<br>"
+            + "<b>P-value:</b> "
+            + str(format(pval, ".4g"))
+            + "<extra></extra>"
+            for param, pval in zip(parameters, p_values)
+        ]
+    )
+    fig.add_shape(
+        {
+            "type": "line",
+            "y0": 0.05,
+            "y1": 0.05,
+            "x0": -0.5,
+            "x1": len(p_values) - 0.5,
+            "xref": "x",
+            "line": {"color": colors["dark gray"], "width": 1.5},
+        }
+    )
+    fig.add_annotation(x=len(p_values) - 0.2, y=0.05, text="P-value<br>= 0.05", showarrow=False)
+    fig = fig.to_dict()
+    fig["layout"].update(
+        barmode="relative",
+        height=500,
+        title=dict(
```

Comment:
Suggest to set hovermode to x to make it easier to hover over low p-value terms
```suggestion
        hovermode="x",
        title=dict(
```